src/test/ui/c-variadic/variadic-ffi-2.stderr

```diff
@@ -1,8 +1,8 @@
-error[E0045]: C-variadic function must have C or cdecl calling convention
-  --> $DIR/variadic-ffi-2.rs:3:11
+error[E0045]: C-variadic function must have a compatible calling convention, like C, cdecl, win64, sysv64 or efiapi
+  --> $DIR/variadic-ffi-2.rs:4:11
    |
 LL | fn baz(f: extern "stdcall" fn(usize, ...)) {
-   |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ C-variadics require C or cdecl calling convention
+   |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ C-variadic function must have a compatible calling convention
```

Comment:
And this would look like

```
error[E0045]: C-variadic function must have a compatible calling convention
   --> $DIR/variadic-ffi-2.rs:4:11
    |
 LL | fn baz(f: extern "stdcall" fn(usize, ...)) {
    |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ C-variadic function must have a compatible calling convention
help: declare the type with a compatible calling convention
   |
 LL | fn baz(f: extern "C" fn(usize, ...)) {
    |                   ~
 LL | fn baz(f: extern "cdecl" fn(usize, ...)) {
    |                   ~~~~~
 LL | fn baz(f: extern "win64" fn(usize, ...)) {
    |                   ~~~~~
 LL | fn baz(f: extern "sysv64" fn(usize, ...)) {
    |                   ~~~~~~
 LL | fn baz(f: extern "efiapi" fn(usize, ...)) {
    |                   ~~~~~~
```